file-upload.js

```diff
@@ -0,0 +1,59 @@
+/*Copyright 2018 Charter DNA Team (CTEC).
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const Minio = require('minio')
+const fs = require('fs');
+
+const file_path = process.env.FILE_PATH
+const accessKey = process.env.ACCESS_KEY
+const secretKey = process.env.SECRET_KEY
+
+const testFolder = "/mnt/brigade/share"
+
+// Gather the date for a timestamp
+const now = new Date();
+var isoString = now.toISOString();
+var date = isoString.split('T')[0];
+var timestamp = date + '-' + now.getHours() + now.getMinutes() + now.getSeconds() + '_'; 
```

Comment:
the next element in the array, `isoString.split('T')[1];` is the time, in lieu of .getHours, .getMinutes....

Reply:
I actually didn't like the visual structure in the iso for the time, so i get the hours minutes seconds out of personal preference.

Reply:
that's ok. `isoString` is UTC, `now` is localTZ. Please use getUTCHours(), getUTCMinutes()...

Reply:
Got ya, makes sense. Changed.